benchmarks/Benchmarks.Trace/Program.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+namespace Benchmarks.Trace
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            Summary summary;
+            summary = BenchmarkRunner.Run<SpanBenchmark>();
```

Comment:
What's the purpose, debugging?

Reply:
Yes, right now the benchmarks doesn't make a lot of sense, I needed to test the exporter. I'm thinking of remove it from the PR, or to change it to maybe tests that make senses.

Reply:
I've changed the benchmark project with only the `SpanBenchmark` in https://github.com/DataDog/dd-trace-dotnet/pull/817/commits/a417528e343732261614e967417cb137479463bd